config/samples/secret-agent_v1alpha1_secretagentconfiguration.yaml

```
@@ -3,25 +3,23 @@ apiVersion: secret-agent.secrets.forgerock.io/v1alpha1
 kind: SecretAgentConfiguration
 metadata:
   name: forgerock-sac
-  
+
 # CRD settings
 spec:
   appConfig:
     createKubernetesObjects: true
 
     # A cloud provider's secret manager is the source of truth.
     # If secrets don't exist, they are generated and stored in the manager.
-    # Azure is 50% complete.
-    secretsManager: none  # none, AWS, or GCP
-    
+    secretsManager: None
```

I think we should keep the sample values around. Also, I believe we look for `none`

```suggestion
    secretsManager: none # none, AWS, Azure, or GCP
```